packages/data-api/package.json

```diff
@@ -17,9 +17,11 @@
     "test": "jest",
     "test:coverage": "yarn test --coverage",
     "install-mv-refresh": "scripts/installMvRefreshModule.sh",
+    "uninstall-mv-refresh": "scripts/uninstallMvRefreshModule.sh",
     "build-analytics-table": "scripts/buildAnalyticsMaterializedView.sh",
-    "refresh-analytics-table": "scripts/refreshAnalyticsTable.sh",
-    "full-rebuild-analytics-table": "scripts/fullRebuildAnalyticsMaterializedView.sh"
+    "drop-analytics-table": "scripts/dropAnalyticsMaterializedView.sh",
+    "full-rebuild-analytics-table": "scripts/fullRebuildAnalyticsMaterializedView.sh",
+    "refresh-analytics-table": "scripts/refreshAnalyticsTable.sh"
```

Comment:
Now to fully reinstall the `mvrefresh` module and analytics table you can just do:
```
yarn workspace @tupaia/data-api drop-analytics-table
yarn workspace @tupaia/data-api uninstall-mv-refresh
yarn workspace @tupaia/data-api install-mv-refresh
yarn workspace @tupaia/data-api build-analytics-table
```
Worth capturing in a single script?

Reply:
Very nice! On the fence about another script, because there are a few here to wade through. I think maybe docs in the readme would be even more useful?